OpenRA.Mods.Common/Activities/Move/Drag.cs

```diff
@@ -19,7 +19,7 @@ public class Drag : Activity
 	{
 		readonly IPositionable positionable;
 		readonly IMove movement;
-		readonly IDisableMove[] moveDisablers;
+		readonly IDisabledTrait disableable;
```

Comment:
Will this scale to the future case when we use upgrades to swap between different movement types?